src/test/java/seedu/address/logic/commands/FilterEntryCommandTest.java

```diff
@@ -74,7 +74,7 @@ public void execute_multipleKeywordsAndSearch_oneEntryFound() {
     }
 
     @Test
-    public void execute_multipleKeywordsOrSearch_noEntryFound() {
+    public void execute_multipleKeywordsAndSearch_noEntryFound() {
```

Comment:
oi lmao, revert back to or leh 
is because if i intended to do an or search, i will get no entry found hahaha